source/designers-guide/theme-php/index.md

```diff
@@ -0,0 +1,286 @@
+---
+layout: default
+title: Theme.php
+github_link: designers-guide/theme-php/index.md
+indexed: true
+---
+
+## Table of contents
+
+-   [Introduction](#introduction)
+-   [Adding JavaScript and CSS](#adding-javascript-and-css)
+-   [Customizing the theme configuration](#customizing-the-theme-configurat)
+-   [Creating a blank theme configuration](#creating-a-blank-theme-configura)
+-   [Configuration sets](#configuration-sets)
+
+## Introduction
+
+The `Theme.php` is the base file of each shopware theme. It contains the basic information like the name, author, description and the license which are displayed in the Theme Manager:
```

Comment:
Please check namings:
- Shopware (the company name with small S, the product with capital S)
- Javascript/CSS/HTML/other things: please make sure that they are consistent across your document and with previously existing doc pages